.travis.yml

```diff
@@ -58,12 +57,37 @@ jobs:
     - DB=mysql
     script:
     - pytest tracim_backend
+  - stage: functional
+    name: Functional tests using Cypress
+    env:
+      - DB=sqlite; TRACIM_DB_FILENAME=tracim_test.sqlite;
+      - TRAVIS_NODE_VERSION="10"
+    before_install:
+      - wget https://sno.phy.queensu.ca/~phil/exiftool/Image-ExifTool-11.11.tar.gz && gzip -dc Image-ExifTool-11.11.tar.gz | tar -xf - && cd Image-ExifTool-11.11 && perl Makefile.PL && sudo make install && cd ..
+    install:
+      - pip install --upgrade pip setuptools
+      - ./setup_default_backend.sh root
+      - ./install_frontend_dependencies.sh root
+      - ./build_full_frontend.sh root
+      - ./setup_functionnal_tests.sh root
+    before_script:
+      - cd backend
+      - sh -c "if [ '$DB' = 'sqlite' ]; then sed -i \"s/\(sqlalchemy.url *= *\).*/\1sqlite:\/\/\/${TRACIM_DB_FILENAME}/\" ${TRACIM_BACKEND_TEST_CONFIG_FILE}; fi"
+      - nohup pserve development.ini&
+      - cd ../functionnal_tests
+
+    script:
+      - $(npm bin)/cypress run
+    after_success:
+      - echo "override"
```

Comment:
Why echo "override" ?

Reply:
This line was a debug line to override Travis config line who call coveralls.